www/js/lib/uiUtil.js

```diff
@@ -746,6 +737,20 @@ function getBrowserLanguage () {
     return language;
 }
 
+// Display The Recently Loaded Article. Called By showReturnLink().
+function viewArticle(){
```

Comment:
Please complete the requirements of codefactor below (if you don't see them, go to the "Files Changed" tab.